src/constants.json

```diff
@@ -118,5 +118,37 @@
     "BID_TARGETING_SET": "targetingSet",
     "RENDERED": "rendered",
     "BID_REJECTED": "bidRejected"
+  },
+  "PREBID_NATIVE_DATA_KEYS_TO_ORTB": {
+    "body": "desc",
+    "body2": "desc2",
+    "sponsoredBy": "sponsored",
+    "cta": "ctatext",
+    "rating": "rating",
+    "address": "address",
+    "downloads": "downloads",
+    "likes": "likes",
+    "phone": "phone",
+    "price": "price",
+    "salePrice": "saleprice",
+    "displayUrl": "displayurl"
+  },  
+  "ASSET_TYPES": {
```

Comment:
I'd put NATIVE somewhere in this name, but maybe it's clearer than it looks to me